ChIP-Seq/tutorials/TAL1_binding_site_identification.md

```diff
@@ -0,0 +1,420 @@
+Identification of the binding sites of the T-cell acute lymphocytic leukemia protein 1 (TAL1) with ChIP-sequencing
+=============
+
+:grey_question: ***Questions***
+
+- *How is raw chIP-seq data processed and analyzed?*
+- *What are the binding sites of Tal1?*
+- *Which genes are regulated by Tal1?*
+
+:dart: ***Objectives***
+
+- *Inspect read quality with FastQC*
+- *Perform read trimming with Trimmomatic*
+- *Align trimmed reads with BWA*
+- *Assess quality and reproducibility of experiments*
+- *Identify Tal1 binding sites with MACS2*
+- *Determine unique/common Tal1 binding sites from G1E and Megakaryocytes*
+- *Identify unique/common Tal1 peaks occupying gene promoters*
+- *Visually inspect Tal1 peaks with Trackster*
+
+:heavy_check_mark: ***Requirements***
+
+- *Galaxy introduction* (add link)
+- *NGS-QC* (add [link](../../NGS-QC/slides/dive_into_qc.md))
+- *NGS-mapping* (add [link](../../NGS-mapping/slides/dive_into_mapping.md))
+- *Trackster* (add link)
```

Comment:
Will these slides be available soon? It seems like the idea is to link tutorials (like ChIP-seq, RNA-seq) to other tutorials (NGS-QC/mapping) in this collection of training materials. Let me know if I'm wrong. I can remove these broken links but keep the NGS* names as placeholders until the indicated materials are available.